onadata/apps/logger/models/submission_review.py

```diff
@@ -51,6 +55,19 @@ class Meta:
         """
         app_label = 'logger'
 
+    @classmethod
+    def set_deleted_at(cls, review_id, deleted_at=timezone.now(), user=None):
```

Comment:
I think we should pick one of `self` or `cls` for use in the same module, for consistency

Reply:
Wouldn't `cls` be more proper to use class methods ?

Reply:
yeah, sorry I had missed that